### PR TITLE
Fix/text locator heatmap

### PR DESCRIPTION
**Pull request by warm-coolguy**
## Summary

- Fix the Heatmap feature. When selecting literature with multiple geometries, they can now be displayed as heatmap again.
- Removed a false warning.

## Instructions for local reproduction and review

Run TextLocator, search for "Health", pick first matching literature result. In the result display, click on the icon for zooming and colouring. The geometry for the river "Eider" should now turn red (since it's the only matching one, and thus matches with max value).

~~The test data is a little boring, I'm requesting additional literature. ~~

Another test: Search for "Runghold" and choose the fictional entry I made up. It's nice for testing and will be used to check the current results in the next fitting weekly.

## Relevant tickets, issues, et cetera

Based/Blocks on https://github.com/Dataport/polar/pull/172

### CODE REVIEW

File: packages/clients/textLocator/src/plugins/GeometrySearch/utils/vectorDisplay.ts [hunk 1]
```diff
@@ -37,26 +37,17 @@ const getZoomFeatures = (features: OlFeature[], item: TreeViewItem | null) =>
...
-          (perTitleAccumulator, toponymToAmount) =>
-            perTitleAccumulator + (toponymToAmount[current.get('title')] || 0),
-          0
-        ),
-    }),
-    {} as Record<string, number>
-  )
-  const max = Math.max(...Object.values(sums))
+  console.error('olFeatures', olFeatures)
+  console.error('locationFrequency', locationFrequency)
```
Comment (dopenguin) [inline packages/clients/textLocator/src/plugins/GeometrySearch/utils/vectorDisplay.ts:43]:
```suggestion
```

File: packages/clients/textLocator/src/plugins/GeometrySearch/utils/vectorDisplay.ts [hunk 2]
```diff
@@ -37,26 +37,17 @@ const getZoomFeatures = (features: OlFeature[], item: TreeViewItem | null) =>
...
+  const max = Math.max(...Object.values(locationFrequency))
+  console.error('mäx', max)
```
Comment (dopenguin) [inline packages/clients/textLocator/src/plugins/GeometrySearch/utils/vectorDisplay.ts:45]:
```suggestion
```

**Review by dopenguin — CHANGES_REQUESTED** [review]
🏓 @warm-coolguy 
Also, a changelog is missing

**Comment by warm-coolguy** [comment]
> 🏓 @warm-coolguy Also, a changelog is missing

🏓 @dopenguin The situation has been resolved by providing a changelog entry matching the implementation. https://github.com/Dataport/polar/pull/195/commits/8f54246843b2320bbc7f8a3cc8404d54b3c1502e

File: packages/clients/textLocator/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,7 @@
 
 - Breaking: Adapt client to new backend API. Previous versions are no longer runnable due the backend API change.
 - Feature: Implement document search, that is, the AddressSearch bar now also offers results for documents and clicking on one will retrieve a list of toponyms from the backend and resolve them against the gazetteer. Previously, we had a "get all documents regarding place" functionality. Now, a "get all places regarding document" feature is implemented. Additionally to the AddressSearch, this can be triggered on documents found in the previously implemented way; the "get all places regarding document" functionality is available from within the GeoSearch result display.
+- Fix: The heatmap feature has been adjusted to match the new data structure and works again.
```
Comment (dopenguin) [inline packages/clients/textLocator/CHANGELOG.md:7]:
```suggestion
- Fix: Adjust heatmap feature to match the new data structure and have it working again.
```

**Review by dopenguin — APPROVED** [review]
🚀  @warm-coolguy 

Also added a suggestion as I prefer imperative in changelogs

**Pull request merged**